pom.xml

```diff
@@ -0,0 +1,416 @@
+      <version>${org.jacoco.maven.plugin.version}</version>
+      <reportSets>
+        <reportSet>
+          <reports>
+            <report>report</report>
+          </reports>
+        </reportSet>
+      </reportSets>
+    </plugin>
+  </plugins>
+</reporting>
+
+<profiles>
+  <profile>
+    <id>minimal</id>
+    <activation>
+      <property>
+        <name>containerjfr.minimal</name>
+        <value>true</value>
+      </property>
+    </activation>
+    <properties>
+      <containerjfr.imageVersion>${project.version}-minimal</containerjfr.imageVersion>
+    </properties>
+  </profile>
+  <profile>
+    <id>non-minimal</id>
+    <activation>
+      <property>
+        <name>containerjfr.minimal</name>
+        <value>!true</value>
+      </property>
+    </activation>
+    <properties>
+      <containerjfr.imageVersion>${project.version}</containerjfr.imageVersion>
+    </properties>
+    <build>
+      <plugins>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>exec-maven-plugin</artifactId>
+          <version>${org.codehaus.mojo.exec.plugin.version}</version>
+          <configuration>
+            <mainClass>com.redhat.rhjmc.containerjfr.ContainerJfr</mainClass>
+            <additionalClasspathElements>
+              <additionalClasspathElement>${project.build.directory}/assets/app/resources</additionalClasspathElement>
+            </additionalClasspathElements>
+          </configuration>
+          <executions>
+            <execution>
+              <id>update-web-client-deps</id>
+              <phase>initialize</phase>
+              <goals>
+                <goal>exec</goal>
+              </goals>
+              <configuration>
+                <executable>npm</executable>
+                <workingDirectory>web-client</workingDirectory>
+                <arguments>
+                  <argument>ci</argument>
+                </arguments>
+              </configuration>
+            </execution>
+            <execution>
+              <id>build-web-client</id>
+              <phase>prepare-package</phase>
+              <goals>
+                <goal>exec</goal>
+              </goals>
+              <configuration>
+                <executable>npm</executable>
+                <workingDirectory>web-client</workingDirectory>
+                <arguments>
+                  <argument>run</argument>
+                  <argument>build</argument>
+                </arguments>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <artifactId>maven-clean-plugin</artifactId>
+          <version>${org.apache.maven.plugins.clean.version}</version>
+          <configuration>
+            <filesets>
+              <fileset>
+                <directory>web-client</directory>
+                <includes>node_modules/**</includes>
+                <includes>dist/**</includes>
+              </fileset>
```

Comment:
This only cleans `dist` for me and not also `node_modules`. I think the directories need to be listed as `<include>` tags within `<includes>`, and not just `<includes>`.